doc/api/util.md

```diff
@@ -655,18 +745,30 @@ util.log('Timestamped message.');
 ```
 
 ### util.print([...])
+<!-- YAML
+added: v0.3.0
+deprecated: v0.11.3
+-->
 
 > Stability: 0 - Deprecated: Use [`console.log()`][] instead.
 
 Deprecated predecessor of `console.log`.
 
 ### util.puts([...])
+<!-- YAML
+added: v0.3.0
+deprecated: v0.11.3
+-->
 
 > Stability: 0 - Deprecated: Use [`console.log()`][] instead.
 
 Deprecated predecessor of `console.log`.
 
 ### util._extend(obj)
```

Comment:
While you are at it, can you fix the markdown formatting here?

This should be `### util.\_extend(obj)`